infrastructure/templates/public-api/application/public-api/publicApiApp.bicep

```diff
@@ -33,6 +33,9 @@ param appInsightsConnectionString string
 @description('Whether to create or update Azure Monitor alerts during this deploy')
 param deployAlerts bool
 
+@description('Enable the Swagger UI')
+param enableSwagger bool = false
```

Comment:
I've taken to removing default values from Bicep files that are only invoked once (e.g. this file is only ever called from `main.bicep`, and `main.bicep` is always providing a value for `enableSwagger`.  And if for some reason it *isn't* providing anything when it really should be, something has gone wrong and we'll find out about it at deploy time.

For lower-level Bicep templates that stand more chance of being reused in various other contexts e.g. `storageAccount.bicep`, then it totally *does* make sense to provide more default values though.

Are you happy with this convention?

Reply:
Cool, that's fine with me! Removed the default now

Reply:
Ta!